### PR TITLE
replace python script to ruby script

### PR DESCRIPTION
**Pull request by bleuchtang**
Python script can't be load directly with gem. I rewrite it.

### CODE REVIEW

File: bin/metrics-uptime.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+#
+# LICENSE:
+# Released under the same terms as Sensu (the MIT license); see LICENSE
+# for details.
+
+require 'sensu-plugin/metric/cli'
+require 'socket'
+
+#
+# Disk Graphite
```
Comment (tas50) [inline bin/metrics-uptime.rb]: This should be updated to the correct plugin name

File: bin/metrics-uptime.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+#
+class UptimeGraphite < Sensu::Plugin::Metric::CLI::Graphite
+  option :scheme,
+         description: 'Metric naming scheme, text to prepend to metric',
+         short: '-s SCHEME',
+         long: '--scheme SCHEME',
+         default: "#{Socket.gethostname}.disk"
```
Comment (tas50) [inline bin/metrics-uptime.rb]: This should be updated for uptime

**Comment by mattyjones** [comment]
@bleuchtang 
Put the script back in please, this would be a breaking change.  When the script is added back in we will merge and make a note in the new version that the python script is depreciated and may be pulled without warning at a later date.  I don;t want to wholesale remove scripts without warning, we need to respect versioning and people's environments.

Thanks!

**Comment by mattyjones** [comment]
@bleuchtang  bump

**Comment by eheydrick** [comment]
Hey @bleuchtang can you update the CHANGELOG for the new plugin. Add it on unreleased.

**Comment by mattyjones** [comment]
@eheydrick are we good here?

File: bin/metrics-uptime.rb (hunk 2 above)
Comment (analytically) [inline bin/metrics-uptime.rb]: This should be fixed before we merge.

**Comment by analytically** [comment]
There's still an issue in the code...

**Pull request merged**